cmds/webboot/webboot.go

```diff
@@ -106,10 +109,14 @@ func (i *ISO) exec(uiEvents <-chan ui.Event, boot bool) error {
 			s := fmt.Sprintf("config.image %s, kernelparams.String() %s", config.image, kernelParams.String())
 			return fmt.Errorf("Booting is disabled (see --dryrun flag), but otherwise would be [%s].", s)
 		}
-		err = bootiso.BootCachedISO(config.image, kernelParams.String() + " waitusb=10")
+		err = bootiso.BootCachedISO(config.image, kernelParams.String()+" waitusb=10")
 	}
 
 	// If kexec succeeds, we should not arrive here
+	if err == nil {
+		err = fmt.Errorf("kexec failed, but gave no error. Be sure to use kexec-tools.")
```

Comment:
This won't help most people, is there some way to make it a bit more informative? This is a tough case to explain to users.

Reply:
Definitely, you are raising the most important point. And this statement is even ~~misleading~~ wrong, since the scenario I tested _was_ actually using kexec-tools, but via cpu. If we cannot tell _what_, I would at least want to tell _where_ the issue was, since the boot mechanism involves so many steps.

Reply:
I've rephrased it to be less suggestive about whether kexec-tools was used and left a TODO.